cluster/config-defaults.yaml

```diff
@@ -947,3 +950,14 @@ sysctl_settings: ""
 
 # enables/disables the minDomains field for pod topology spread.
 min_domains_in_pod_topology_spread_enabled: "true"
+
+# Set defaults values to be used by skipper ingress when the open policy agent is enabled
+{{if eq .Cluster.Environment "test"}}
+skipper_open_policy_agent_bucket_arn: "arn:aws:s3:::zalando-open-policy-agent-sandbox-bundles"
+skipper_open_policy_agent_observability_url: "https://open-policy-agent-sandbox-observability.corporate-iam.zalan.do/v1"
+skipper_open_policy_agent_bundles_url: "https://zalando-open-policy-agent-sandbox-bundles.s3.eu-central-1.amazonaws.com"
+{{else}}
+skipper_open_policy_agent_bucket_arn: "arn:aws:s3:::zalando-open-policy-agent-prod-bundles"
+skipper_open_policy_agent_observability_url: "https://open-policy-agent-observability.corporate-iam.zalan.do/v1"
+skipper_open_policy_agent_bundles_url: "https://zalando-open-policy-agent-prod-bundles.s3.eu-central-1.amazonaws.com"
```

Comment:
Let's also define a "default" value for token as empty string `""`, similar to https://github.com/zalando-incubator/kubernetes-on-aws/blob/a06e8318aef6758a1d7a0987a3e8ca0b21fe7323/cluster/config-defaults.yaml#L861

This way all possible config items are documented in this file.